hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -203,6 +207,27 @@ private void encodeAndWriteParityCells() throws IOException {
     }
   }
 
+  private void logStreamError(List<ECBlockOutputStream> failedStreams,
+                              String operation) {
```

Comment:
why do we need to collect into map? is there a way to collect into set? seems like we have not used values right?